src/material/tabs/paginated-tab-header.ts

```diff
@@ -246,6 +246,30 @@ export abstract class MatPaginatedTabHeader
     });
   }
 
+  /** A method responsible for sending any change that could affect layout about
+   * items to subscribers.
+   */
+  _getItemChanges() {
+    const itemsTracked = new Set<MatPaginatedTabHeaderItem>();
+    const itemContainerItemChanged = new Subject<void>();
+
+    const observer = new ResizeObserver(() => {
+      itemContainerItemChanged.next();
+    });
+
+    return merge(this._items.changes, itemContainerItemChanged).pipe(
+      startWith(this._items),
+      map(() => {
+        for (const item of this._items.toArray()) {
+          if (!itemsTracked.has(item)) {
+            observer.observe(item.elementRef.nativeElement);
```

Comment:
We should run the observer outside of the `NgZone`.

Reply:
Done.

Reply:
The call above should be `runOutsideAngular`, not `run`.

Reply:
Done. Thanks for the tip.